src/neptune/exceptions.py

```diff
@@ -530,11 +530,11 @@ def __init__(self, container_type: ContainerType, callback_name: str):
 {end}
 Read-only mode can be used only with an existing {container_type}.
 
-The {python}{container_type}{end} parameter of {python}{callback_name}{end} must be provided and reference
-an existing run when using {python}mode="read-only"{end}.
+Pass the ID of a {container_type} to the {python}with_id{end} argument of {python}{callback_name}{end}
+when using {python}mode="read-only"{end}.
 
 You may also want to check the following docs pages:
-    - https://docs.neptune.ai/logging/to_existing_object/
+    - https://docs.neptune.ai/usage/resuming_run/
```

Comment:
Page does not (yet) exist :)

Reply:
Good catch. It's misspelled because in my haste I deviated from my normal scheme of naming files 🙊